virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -538,3 +538,35 @@ def create_remote_disk_by_same_metadata(vm, params):
 
     remote_session.close()
     return blk_source
+
+
+def fill_sth_in_vm(vm, target, size_value=500):
```

Comment:
I think it is better to use "fill_null_in_vm" as function name for better understanding

Reply:
updated